solr/core/src/java/org/apache/solr/servlet/LoadAdminUiServlet.java

```diff
@@ -15,6 +15,13 @@
  * limitations under the License.
  */
 package org.apache.solr.servlet;
+import javax.servlet.http.HttpServletRequest;
```

Comment:
nit: did you mean to move these imports around?

Reply:
Yes. I did that because the precommit failed before locally.